eahub/profiles/models.py

```diff
@@ -153,6 +154,17 @@ class OrganisationalAffiliation(enum.Enum):
         WILD_ANIMAL_INITIATIVE: "Wild Animal Initiative"
     }
 
+def create_pretty_property_list(propertyClass, standard_list, other_list):
+    pretty_list = ''
+    if standard_list:
+        pretty_list += ", ".join(map(propertyClass.label, standard_list))
+    if other_list:
+        pretty_list = pretty_list + ', ' + other_list if standard_list else other_list
+    if (standard_list and other_list) is False:
```

Comment:
`if not (standard_list and other_list):` (`is` in Python compares object identity and I'm not sure that does the right thing here)

Reply:
Didn't implement as it change the functions behaviour (returns N/A when it shouldn't)

Reply:
This is definitely suspicious and I remain confused about how it works, but we can leave it for now.